contracts/BlastPoints.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.20;
+
+import {IBlastPoints} from "./interfaces/IBlastPoints.sol";
+
+/**
+ * @title BlastPoints
+ * @notice This contract is a base contract for future contracts that wish to claim Blast points to inherit from
```

Comment:
Blast points cannot be claimed by contracts, it's just given to us by Blast and we distribute them to our users

Reply:
https://github.com/LooksRare/contracts-libs/pull/103/commits/96c81ecce1ffb035e5749f47a16025cf66d7b328